### ISSUE TITLE
Cannot parse schema prefix

### ISSUE THREAD

**Issue by hieulw** [xnuinside/omymodels#5]
Hello, this is a very useful project. It would be nice if you can parse the schema in DDL file.
```sql
CREATE TABLE "my-custom-schema"."users" (
  "id" SERIAL PRIMARY KEY,
  "name" varchar,
  "created_at" timestamp,
  "updated_at" timestamp,
  "country_code" int,
  "default_language" int
);
```

Thanks for a nice project!

### Updated:
`"schema-name"."table-name"` works fine, but `"schema--name"."table--name"` ( with double `-` symbol) doesn't work

**Comment by xnuinside** [comment]
Hello! This is strange, because schema is supported, here is an example in tests https://github.com/xnuinside/omymodels/blob/main/tests/test_models_with_index.py . But I have an idea, that maybe problem in ‘-‘ symbol. I don’t remember that I added anything with it in tests for parser. I will back from vacation on 31 March and anyway will check it and fix :)

**Comment by hieulw** [comment]
Thanks,
I test again and this is what I found.
"schema-name"."table-name" works fine, but "schema--name"."table--name" ( with double - symbol) doesn't work

**Comment by xnuinside** [comment]
@hieulw got it! thank you for testing. When I will be back - I will fix it.

**Comment by xnuinside** [comment]
@hieulw I just tested your first DDL example and got valid output with last version 0.7.3:

from gino import Gino

db = Gino(schema="my-custom-schema")


class Users(db.Model):

    __tablename__ = 'users'

    id = db.Column(db.Integer(), autoincrement=True, nullable=False, primary_key=True)
    name = db.Column(db.String())
    created_at = db.Column(db.TIMESTAMP())
    updated_at = db.Column(db.TIMESTAMP())
    country_code = db.Column(db.Integer())
    default_language = db.Column(db.Integer())

**Comment by xnuinside** [comment]
@hieulw fixed in 0.4.1, also important change - create_gino_models renamed to create_models, because now library also allow to generate Pydantic models :)
